stubs/core.stub

```diff
@@ -315,16 +321,16 @@ function is_callable(mixed $value, bool $syntax_only = false, ?string &$callable
 function abs($num) {}
 
 /**
- * @return ($string is lowercase-string ? lowercase-string : string)
+ * @return ($string is lowercase-string ? lowercase-string : ($string is uppercase-string ? uppercase-string : string))
  */
 function trim(string $string, string $characters = " \n\r\t\v\x00"): string {}
 
 /**
- * @return ($string is lowercase-string ? lowercase-string : string)
+ * @return ($string is lowercase-string ? lowercase-string : ($string is uppercase-string ? uppercase-string : string))
  */
 function ltrim(string $string, string $characters = " \n\r\t\v\x00"): string {}
 
 /**
- * @return ($string is lowercase-string ? lowercase-string : string)
+ * @return ($string is lowercase-string ? lowercase-string : ($string is uppercase-string ? uppercase-string : string))
```

Comment:
Same, should check `$string is lowercase-string&uppercase-string` first.

Reply:
Resolved in 2f78f966a